trigger/netdevices/__init__.py

```diff
@@ -937,13 +937,13 @@ def map_attr(attr):
             for attr, val in kwargs.iteritems():
```

Comment:
GH won't let me note on it, but L935's comment no longer applies.

Other than that, ship it!